src/stratigraphy/annotations/draw.py

```diff
@@ -138,10 +138,10 @@ def draw_metadata(
     """
     # TODO associate correctness with the extracted coordinates in a better way
     coordinate_color = "green" if is_coordinate_correct else "red"
-    coordinate_rect = fitz.Rect([5, 5, 200, 25])
+    coordinate_rect = fitz.Rect([5, 5, 250, 30])
```

Comment:
I needed to increase the size of the boxes; otherwise, the coordinates would not be printed because the box was too small.